llvm/lib/Target/X86/X86InstrCompiler.td

```diff
@@ -1864,62 +1864,37 @@ def : Pat<(fshl GR64:$src1, GR64:$src2, (shiftMask64 CL)),
 def : Pat<(fshr GR64:$src2, GR64:$src1, (shiftMask64 CL)),
           (SHRD64rrCL GR64:$src1, GR64:$src2)>;
 
-let Predicates = [HasBMI2] in {
+multiclass bmi_shift_mask_patterns<SDNode op, string name, string Suffix = ""> {
```

Comment:
Suffix -> suffix

Reply:
No need to pass "name" as an argument, you only need to use "defm SARX :" at line 1889.